test/e2e/storage/persistent_volumes-local.go

```diff
@@ -105,43 +110,286 @@ var _ = framework.KubeDescribe("[Volume] PersistentVolumes-local [Feature:LocalP
 			testVol = nil
 		})
 
-		It("should be able to mount and read from the volume", func() {
+		It("should be able to mount and read from the volume using one-command containers", func() {
 			By("Initializing test volume")
 			testVol = setupLocalVolume(config, node)
 
 			By("Creating local PVC and PV")
 			createLocalPVCPV(config, testVol)
 
-			By("Creating a pod to consume the PV")
-			readCmd := fmt.Sprintf("cat /mnt/volume1/%s", testFile)
-			podSpec := createLocalPod(config, testVol, readCmd)
-			f.TestContainerOutput("pod consumes PV", podSpec, 0, []string{testContents})
+			By("Creating a pod to read from the PV")
+			_, readCmd := createWriteAndReadCmds(volumeDir, testFile, "" /*writeTestFileContent*/)
```

Comment:
L428 in `runLocalUtil`: there;s a const for the busybox image name in framework.